package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.0.7"
+    "nan": "^2.0.7"
```

Comment:
This is not a good way to specify dependencies. Please use exact versions to avoid breaking changes being introduced.